pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -680,7 +680,7 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 				LoadDistribution: loadDistribution,
 				FrontendPort:     to.Int32Ptr(port.Port),
 				BackendPort:      to.Int32Ptr(port.Port),
-				EnableFloatingIP: to.BoolPtr(true),
+				EnableFloatingIP: to.BoolPtr(to.Int32Ptr(port.Port) == to.Int32Ptr(port.Port)),
```

Comment:
looks like `to.Int32Ptr(port.Port) == to.Int32Ptr(port.Port)` should be always true here?

Reply:
I'm sorry, I understand it wrong, I will close it, from now on it is correct.